### PR TITLE
Fit to window regime

### LINKED ISSUE #51: Fit to Window 3: Revenge of the Sith
We should make "Fit to Window" menu action works as regime not just zoom action. For example look how Preview.app on macOS works

### PR DESCRIPTION
**Pull request by hombit**
Fix #51

Check this GUI features before merge:

- [x] Positive and negative `BITPIX` images
- [x] Zoom, rotate and flip
- [x] Level control
- [x] Color maps
- [x] Open new image in the same window and in new window

### CODE REVIEW

**Comment by matwey** [comment]
I cannot uncheck menu item on Linux. "Fit to window" is always enabled for me.

File: include/openglwidget.h [hunk 1]
```diff
@@ -137,9 +137,11 @@ class OpenGLWidget: public QOpenGLWidget, protected QOpenGLFunctions {
 	openGL_unique_ptr<AbstractOpenGLPlan> plan_;
 private:
 	Viewrect viewrect_;
+	bool fitted_ = false;
```
Comment (matwey) [inline include/openglwidget.h]: Could you please initialize variables in constructor?

**Comment by matwey** [comment]
Overall Linux experience is terrifying. Everything is flickering. We have to discuss it on Monday.

**Comment by matwey** [comment]
If fit to window is a regime, then should zoom still work when fit to window is enabled?

**Comment by hombit** [comment]
Zooming turns off “fit to window”. I think the menu item should be inactive when it is checked, just like Preview.app does

File: include/openglwidget.h (hunk 1 above)
Comment (hombit) [reply include/openglwidget.h]: Ok

**Comment by hombit** [comment]
The behaviour of "Fit to window" now is: when it is checked, it is marked ✔︎ and it can be unchecked manually. If user zooms, then this regime if switched off.

**Comment by matwey** [comment]
Fixed via #149

**Pull request closed without merging**